CONTRIBUTING.md

```diff
@@ -0,0 +1,217 @@
+  `vNalpha` (with the deprecated field) that go-control-plane uses internally. The PR author deprecating the old configuration is responsible for
+  updating all tests and canonical configuration. The majority of tests and
+  configuration for a feature should be expressed in terms of the latest go-control-plane internal
+  configuration (i.e. `vNalpha`).
+* We will delete deprecated configuration across major API versions. E.g. a field marked deprecated
+  in v2 will be removed in v3.
+
+## Submitting a PR
+
+* Fork the repo.
+* Create your PR.
+* Tests will automatically run for you.
+* We will **not** merge any PR that is not passing tests.
+* PRs are expected to have 100% test coverage for added code. This can be verified with a coverage
+  build. If your PR cannot have 100% coverage for some reason please clearly explain why when you
+  open it.
+* Any PR that changes user-facing behavior **must** have associated documentation in [docs](docs) as
+  well as the [changelog](CHANGELOG.md).
+* All code comments and documentation are expected to have proper English grammar and punctuation.
+  If you are not a fluent English speaker (or a bad writer ;-)) please let us know and we will try
+  to find some help but there are no guarantees.
+* Your PR title should be descriptive, and generally start with a subsystem name followed by a
+  colon. Examples:
+  * "docs: fix grammar error"
+  * "http conn man: add new feature"
+* Your PR commit message will be used as the commit message when your PR is merged. You should
+  update this field if your PR diverges during review.
+* Your PR description should have details on what the PR does. If it fixes an existing issue it
+  should end with "Fixes #XXX".
+* If your PR is co-authored or based on an earlier PR from another contributor,
+  please attribute them with `Co-authored-by: name <name@example.com>`. See
+  GitHub's [multiple author
+  guidance](https://help.github.com/en/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors)
+  for further details.
+* When all of the tests are passing and all other conditions described herein are satisfied, a
+  maintainer will be assigned to review and merge the PR.
+* Once you submit a PR, *please do not rebase it*. It's much easier to review if subsequent commits
+  are new commits and/or merges. We squash rebase the final merged commit so the number of commits
+  you have in the PR don't matter.
+* We expect that once a PR is opened, it will be actively worked on until it is merged or closed.
+  We reserve the right to close PRs that are not making progress. This is generally defined as no
+  changes for 7 days. Obviously PRs that are closed due to lack of activity can be reopened later.
+  Closing stale PRs helps us to keep on top of all of the work currently in flight.
+
+## PR review policy for maintainers
+
+* Typically we try to turn around reviews within one business day.
```

Comment:
I am not sure we can make this promise. We don't have enough maintainers to provide an SLO.

Reply:
Yup I wasn't sure what I should put there so I left it, but I'll omit